src/renderer/entities/wallet/ui/CheckPermission/CheckPermission.tsx

```diff
@@ -0,0 +1,26 @@
+import { PropsWithChildren } from 'react';
+
+import { Account, Wallet } from '@shared/core';
+import { OperationType } from '../../common/types';
+import { getOperationTypeFn } from '../../common/utils';
+
+type Props = {
+  operationType: OperationType;
+  wallet?: Wallet;
```

Comment:
This application state looks confusing and moreover incorrect
`wallet` & `accounts` both exist or don't exist at the same time

Reply:
`$activeAccounts` always has type `Account[]` and we don't  need to check it